Could not get unknown property 'bintray.user' for project ':lib' of type org.gradle.api.Project

Executing gradlew within the capillary directory gradle fails with the following error:

```
D:\capillary>gradlew tasks

FAILURE: Build failed with an exception.

* Where:
Script 'D:\capillary\lib\publish.gradle' line: 56

* What went wrong:
A problem occurred evaluating script.
> Could not get unknown property 'bintray.user' for project ':lib' of type org.gradle.api.Project.
```

### Steps to reproduce
```
git clone https://github.com/google/capillary.git
cd capillary
gradlew tasks
```

### Environment
Windows10, Oracle Java 9 64bit

--- a/lib-android/publish.gradle
+++ b/lib-android/publish.gradle
@@ -24,8 +24,10 @@ install {
             }
             developers {
                 developer {
-                    name project.property('user.name')
-                    email project.property('user.email')
+                    name project.hasProperty('user.name') ?
+                            project.property('user.name') : System.getenv('USER_NAME')
+                    email project.hasProperty('user.email') ?
+                            project.property('user.email') : System.getenv('USER_EMAIL')
                     organization = 'Google LLC'
                     organizationUrl 'https://www.google.com'
                 }
@@ -40,8 +42,10 @@ install {
 }
 
 bintray {
-    user = project.property('bintray.user')
-    key = project.property('bintray.apikey')
+    user = project.hasProperty('bintray.user') ?
+            project.property('bintray.user') : System.getenv('BINTRAY_USER')
+    key = project.hasProperty('bintray.apikey') ?
+            project.property('bintray.apikey') : System.getenv('BINTRAY_APIKEY')
     configurations = ['archives']
     pkg {
         repo = bintrayRepo

--- a/lib/publish.gradle
+++ b/lib/publish.gradle
@@ -53,8 +53,10 @@ publishing {
 }
 
 bintray {
-    user = project.property('bintray.user')
-    key = project.property('bintray.apikey')
+    user = project.hasProperty('bintray.user') ?
+            project.property('bintray.user') : System.getenv('BINTRAY_USER')
+    key = project.hasProperty('bintray.apikey') ?
+            project.property('bintray.apikey') : System.getenv('BINTRAY_APIKEY')
     publications = ['MavenLib']
     pkg {
         repo = bintrayRepo

